addons/website_slides/models/slide_question.py

```diff
@@ -28,6 +29,19 @@ def _check_answers_integrity(self):
             if len(question.answer_ids) < 2:
                 raise ValidationError(_('Question "%s" must have 1 correct answer and at least 1 invalid answer', question.question))
 
+    @api.depends('slide_id')
+    def _recompute_attempts_count(self):
+        slide_partners = self.env['slide.slide.partner'].sudo().search([('slide_id', 'in', self.slide_id.ids)])
+        slide_stats = dict((s.slide_id.id, dict({'attempts_count': 0})) for s in slide_partners)
+
+        for slide_partner in slide_partners:
+            slide_stats[slide_partner.slide_id.id]['attempts_count'] += slide_partner.quiz_attempts_count
+
+        for question in self:
+            stats = slide_stats.get(question.slide_id.id)
+            attempts_count = stats.get('attempts_count', 0) if stats else 0
+            question.write({'attempts_count': attempts_count})
+
```

Comment:
We could factorise a bit this one and compute_statistics method.
Move content of _compute_statistics into _recompute method and in the compute method, call recompute giving field names to recompute in params.

Reply:
PR updated :)